frontend/package.json

```
@@ -40,5 +41,6 @@
       "last 1 firefox version",
       "last 1 safari version"
     ]
-  }
+  },
+  "proxy": "http://localhost:8080/"
```

Does this work with the slash?
Just wondering.
I used mine without the slash.

```suggestion
  "proxy": "http://localhost:8080"
```